episodes/05-forks.md

```diff
@@ -324,14 +324,14 @@ sequenceDiagram
     GHM -->> M: #f
     Note right of GHM: git pull
     Note over F: Delete branch:<br/>git branch -d feature
-    box Upstream Remote - GitHub
+    box Met Office Repo - GitHub
     participant UM
     end
-    box Fork Remote - GitHub
+    box Your Fork - GitHub
     participant GHM
     participant GHF
     end
-    box Fork - Local
+    box Your Fork - Local
```

Comment:
The fork is only the GitHub hosted repo, whereas the local one is a clone or working copy
```suggestion
    box Your Working Copy - Local
```